Fixtures/ClangModules/Cpp/Sources/cwrapper/cwapper.cpp

```diff
@@ -0,0 +1,7 @@
+#import "include/cwrapper.h"
+#import "cpplib.h"
+
+int cwrapperfive() {
```

Comment:
Despite appearances, this is *not* actually a C function, but rather a C++ function because it is in a C++ source file, with its symbol name mangled as such. This is the cause of the linked error. To make this be compiled and exported as a C function, it needs to be annotated with `extern "C"`